examples/src/main/java/org/spine3/examples/aggregate/ClientApp.java

```diff
@@ -0,0 +1,246 @@
+        public void onCompleted() {
+            log().info("Subscription update delivery completed.");
+        }
+    };
+
+
+    /** Construct the client connecting to server at {@code host:port}. */
+    public ClientApp(String host, int port) {
+        commandFactory = CommandFactory.newBuilder()
+                                       .setActor(newUserId(Identifiers.newUuid()))
+                                       .setZoneOffset(ZoneOffsets.UTC)
+                                       .build();
+        channel = ManagedChannelBuilder
+                .forAddress(host, port)
+                .usePlaintext(true)
+                .build();
+        blockingClient = CommandServiceGrpc.newBlockingStub(channel);
+        subscriptionClient = SubscriptionServiceGrpc.newStub(channel);
+    }
+
+    private Command createOrder(OrderId orderId) {
+        final CreateOrder msg = CreateOrder.newBuilder()
+                                           .setOrderId(orderId)
+                                           .build();
+        return commandFactory.create(msg);
+    }
+
+    private Command addOrderLine(OrderId orderId) {
+        final int bookPriceUsd = 52;
+        final Book book = Book.newBuilder()
+                              .setBookId(BookId.newBuilder()
+                                               .setISBN("978-0321125217")
+                                               .build())
+                              .setAuthor("Eric Evans")
+                              .setTitle("Domain Driven Design.")
+                              .setPrice(newMoney(bookPriceUsd, USD))
+                              .build();
+        final int quantity = 1;
+        final Money totalPrice = newMoney(bookPriceUsd * quantity, USD);
+        final OrderLine orderLine = OrderLine.newBuilder()
+                                             .setProductId(AnyPacker.pack(book.getBookId()))
+                                             .setQuantity(quantity)
+                                             .setPrice(totalPrice)
+                                             .build();
+        final AddOrderLine msg = AddOrderLine.newBuilder()
+                                             .setOrderId(orderId)
+                                             .setOrderLine(orderLine)
+                                             .build();
+        return commandFactory.create(msg);
+    }
+
+    private Command payForOrder(OrderId orderId) {
+        final BillingInfo billingInfo = BillingInfo.newBuilder()
+                                                   .setInfo("Payment info is here.")
+                                                   .build();
+        final PayForOrder msg = PayForOrder.newBuilder()
+                                           .setOrderId(orderId)
+                                           .setBillingInfo(billingInfo)
+                                           .build();
+        return commandFactory.create(msg);
+    }
+
+
```

Comment:
To be processed along with `examples` project update after `core-java` 0.6.0 release.

I have adjusted the IDEA code style configuration. The commit can be found [here](https://github.com/SpineEventEngine/core-java/commit/dc4dbaaffa5403a7330e950a7443dab35d7a44ea).